data/mods/DinoMod/monstergroups/dinosaur.json

```diff
@@ -4,11 +4,10 @@
     "type": "monstergroup",
     "name": "GROUP_DINOSAUR",
     "monsters": [
-      { "monster": "mon_null", "weight": 614 },
```

Comment:
If you remove "mon_null" doesn't that mean that anytime a monstergroup is called it will always produce a monster?  Drastically increasing the number of monsters called from current spawn rates in some cases?